tests/test_converter.py

```diff
@@ -1,6 +1,6 @@
+import pytest
```

Comment:
The original position of the `purest` import was correct, since pytest is a third-party module. Here's the PEP8 pattern:

```
stdlib import
stdlib import
stdlib import

third party import
third party import
third party import

internal import
internal import
```

Reply:
oh shoot, thought it was stdlib